packages/devextreme-cli/testing/creating.js

```diff
@@ -0,0 +1,31 @@
+const reactEnv = require('./env.react');
+const vueEnv = require('./env.vue');
+const angularEnv = require('./env.angular');
+const minimist = require('minimist');
+
+const args = minimist(process.argv.slice(), {
+    default: {
+        template: 'all'
+    },
+    alias: {
+        t: 'template'
```

Comment:
Why this argument named so? Previously it was a [testNamePattern](https://jestjs.io/docs/en/cli#--testnamepatternregex)  flag that was passed to `jest`. It looks like it has another meaning now.